docs/reference/naming.md

```diff
@@ -84,61 +84,85 @@ defs.GetServiceByProto('DNS','udp')
 
 ## Methods
 
-```
-**GetIpParents(self, query)**
-> Return network tokens that contain IP in query.
-> Args:
-> > query: an ip string ('10.1.1.1') or nacaddr.IP object
-> Returns:
-> > rval2: a list of tokens containing this IP
-**GetNet(self, query)**
-> Expand a network token into a list of nacaddr.IP objects.
-> Args:
-> > query: Network definition token which may include comment text
-> Raises:
-> > BadNetmaskTypeError: Results when an unknown netmask\_type is
-> > specified.  Acceptable values are 'cidr', 'netmask', and 'hostmask'.
-> Returns:
-> > List of nacaddr.IP objects
-> Raises:
-> > UndefinedAddressError: for an undefined token value
-**GetNetAddr(self, token)**
-> Given a network token, return a list of nacaddr.IP objects.
-> Args:
-> > token: A name of a network definition, such as 'INTERNAL'
-> Returns:
-> > A list of nacaddr.IP objects.
-> Raises:
-> > UndefinedAddressError: if the network name isn't defined.
-**GetService(self, query)**
-> Given a service name, return a list of associated ports and protocols.
-> Args:
-> > query: Service name symbol or token.
-> Returns:
-> > A list of service values such as ['80/tcp', '443/tcp', '161/udp', ...]
-**GetServiceByProto(self, query, proto)**
-> Given a service name, return list of ports in the service by protocol.
-> Args:
-> > query: Service name to lookup.
-> > proto: A particular protocol to restrict results by, such as 'tcp'.
-> Returns:
-> > A list of service values of type 'proto', such as ['80', '443', ...]
-**GetServiceParents(self, query)**
-> Given a service, return any tokens containing the value.
-> Args:
-> > query: a service or token name, such as 53/tcp or DNS
-> Returns:
-> > rval2: a list of tokens that contain query or parents of query
-**ParseNetworkList(self, data)**
-> Take an array of network data and import into class.
-> This method allows us to pass an array of data that contains network
-> definitions that are appended to any definitions read from files.
-> Args:
-> > data: array of text lines containing net definitions.
-**ParseServiceList(self, data)**
-> Take an array of service data and import into class.
-> This method allows us to pass an array of data that contains service
-> definitions that are appended to any definitions read from files.
-> Args:
-> > data: array of text lines containing service definitions.
-```
+### Network Query Methods
+
+#### Naming.GetNet(query: str) -> List[Union[IPv4, IPv6]]
+::: aerleon.lib.naming.Naming.GetNet
```

Comment:
Does this render correctly? When I go to view file it doesn't seem to.

Reply:
Yes, this is using mkdocs codegen. It pulls the docstrings in to the text.